testsuite/run_sets/refhost.yml

```diff
@@ -20,8 +20,8 @@
 - features/init_clients/sle_client.feature
 - features/init_clients/sle_minion.feature
 - features/init_clients/sle_ssh_minion.feature
-- features/init_clients/centos_ssh_minion.feature
-- features/init_clients/ubuntu_ssh_minion.feature
+- features/init_clients/centos_minion.feature
```

Comment:
We should also rename both feature files named as min_centos/ubuntu_salt.feature

Reply:
Done.  Renamed as:
- testsuite/features/init_clients/min_centos_salt.feature
- testsuite/features/init_clients/min_ubuntu_salt.feature
- testsuite/features/secondary/min_centos_ssh.feature
- testsuite/features/secondary/min_ubuntu_ssh.feature